src/components/popover/tab/keypress/keypress-tab.component.ts

```diff
@@ -16,6 +16,8 @@ export class KeypressTabComponent implements OnChanges, Tab {
     @Input() defaultKeyAction: KeyAction;
     @Input() longPressEnabled: boolean;
 
+    @Output() validAction = new EventEmitter();
```

Comment:
But we need to trigger validator every single time that user do an action, so we need some kind of a event emitter.